app/src/main/java/org/zalando/nakadi/config/NakadiConfig.java

```diff
@@ -30,11 +28,6 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(NakadiConfig.class);
 
-    @Bean
-    public TaskExecutor taskExecutor() {
-        return new SimpleAsyncTaskExecutor();
```

Comment:
what does Spring use instead by default to manage threads?

Reply:
Good point — I wanted to check that, but got sidetracked :P

Looks like it uses the same one: https://docs.spring.io/spring-framework/docs/current/javadoc-api/org/springframework/web/context/request/async/WebAsyncManager.html#setTaskExecutor-org.springframework.core.task.AsyncTaskExecutor-

> By default a SimpleAsyncTaskExecutor instance is used.